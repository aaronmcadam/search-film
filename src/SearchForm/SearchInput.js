import styled from 'styled-components';

const SearchInput = styled.input`
  border-radius: 10px;
  font-size: 1.5rem;
  margin-right: 10px;
  padding: 10px;
`;

export default SearchInput;
