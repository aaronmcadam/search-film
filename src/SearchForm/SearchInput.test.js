import React from 'react';
import { shallow } from 'enzyme';
import SearchInput from './SearchInput';

const setup = () => {
  const wrapper = shallow(<SearchInput />);

  return {
    wrapper,
  };
};

test('renders', () => {
  const { wrapper } = setup();

  expect(wrapper).toMatchSnapshot();
});
