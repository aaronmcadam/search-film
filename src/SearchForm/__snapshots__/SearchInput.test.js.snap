// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<input
  className="sc-bdVaJa edzqSS"
/>
`;
